src/helpers/utils.ts

```diff
@@ -247,3 +247,16 @@ async function updateWalletConnectWhitelist(
 
   return true;
 }
+
+export function getSpaceController(space: string) {
+  const shibariumNetworkMap = {
+    '1': '109',
+    '11155111': '157'
+  };
+
+  return snapshot.utils.getSpaceController(
+    space,
+    space.endsWith('.shib') ? shibariumNetworkMap[DEFAULT_NETWORK] : DEFAULT_NETWORK,
+    { broviderUrl }
+  );
+}
```

Comment:
Better to use `process.env.NETWORK === 'testnet'` to find if it is testnet or not, and to select shibariumNetwork based on that

just to avoid  hardcoding of `11155111` here 


or we can use `SNAPSHOT_ENV` in `src/writer/settings.ts` to pass it here

Reply:
else looking good